src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -132,7 +133,7 @@ export function createMessageContent(
         addReplyToMessageContent(content, replyToEvent, {
             permalinkCreator,
             includeLegacyFallback: includeReplyLegacyFallback,
-            inThread: relation?.rel_type === RelationType.Thread,
+            inThread: relation?.rel_type === THREAD_RELATION_TYPE.name,
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?